designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -133,6 +133,12 @@ class ProcessesResources(
             processService.getProcessActions(processId.id)
           }
         }
+      } ~ path("processes" / ProcessNameSegment / "lastlyDeployedVersionId") { processName =>
```

Comment:
I might be just picking on this but `lastly` does not sound right for me in this context. What about `last` -> `lastDeployedVersionId`? And similar change in classes ans usages. You can ignore it if you will :upside_down_face:

Reply:
Yeahh, that sounds much better :)